src/classes/CON_PreferredPhone_TEST.cls

```diff
@@ -376,32 +349,23 @@ private class CON_PreferredPhone_TEST {
 
     @isTest 
     static void testBatchWPreferredPhoneSelectionMisMatch() {
-         
-        UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Disable_Preferred_Phone_Enforcement__c = True)
-            );
-            
-        
+        disablePreferredPhone();
+
         Contact newCont = new Contact(
             FirstName = 'JohnnyTest1',
             LastName = 'JohnnyTest',
-            OtherPhone  = '0000000000',
+            OtherPhone = '0000000000',
             MobilePhone  = '0000000000'
         );
         insert newCont;
         
-        
         Test.startTest();
-            UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Disable_Preferred_Phone_Enforcement__c = False)
-            );
-                
-            UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Preferred_Phone_Selection__c = 'homePhone')
-            );
+            Hierarchy_Settings__c settings = enablePreferredPhone();
```

Comment:
oh nice, I kinda like the indentation here :) Gonna have to steal that from you and @skosgi1 :)

Reply:
Yep, I learned it from him! It's not technically a "block" of code but it makes it more clear what code is in between start/stopTest